src/nbody_base.jl

```diff
@@ -207,7 +185,41 @@ function pairwise_lennard_jones_acceleration(rs,
     return -24*system.lj_parameters.ϵ/system.lj_parameters.σ*accel
 end
 
-function temperature(result :: SimulationResult, time :: Float64)
+function get_velocity(result :: SimulationResult, time, i = 0)
```

Comment:
this doesn't look right

Reply:
oh, probably not done here yet.

Reply:
I am confused, what did you mean? I have two variants:
1) `time` in `get_velocity` is without type assertion
2) `i` in `get_velocity` represents particle for which I obtain the velocity. And if i=0, then get velocity values for all the particles in a system - probably it is better to have two specific functions.

Reply:
Also, the name `result` for a function argument is a bad choice.